inc/saiarsprofile.h

```diff
@@ -176,155 +176,219 @@ typedef enum _sai_ars_profile_attr_t
 
     /**
      * @brief Number of quantization bands support for quality map.
+     * Count will be 2 if SAI_ARS_PROFILE_ATTR_PORT_LOAD_PAST and SAI_ARS_PROFILE_ATTR_PORT_LOAD_FUTURE are enabled.
      *
-     * @type sai_uint8_t
+     * @type sai_u8_list_t
      * @flags READ_ONLY
      */
     SAI_ARS_PROFILE_ATTR_QUANT_BANDS,
 
     /**
-     * @brief Minimum threshold used for quantization process in Mbps for band 0.
+     * @brief Minimum threshold used for quantization.
+     * Count will be 2 if SAI_ARS_PROFILE_ATTR_PORT_LOAD_PAST and SAI_ARS_PROFILE_ATTR_PORT_LOAD_FUTURE are enabled.
      *
-     * @type sai_uint32_t
-     * @flags CREATE_AND_SET
-     * @default 0
+     * @type sai_u32_list_t
+     * @flags READ_ONLY
```

Comment:
Syntax wise suggestions:

1.  not backward compatible.
2.  Consider leaving these attrs as is and instead introduce:

    /\*\*     \* @brief Minimum thresholds used for quantization process in Mbps for Past load.     

   .  \*     \* @type sai\_uint32\_list\_t     

            \* @flags READ\_ONLY     

             \* @default 0     \*/  

  SAI\_ARS\_PROFILE\_ATTR\_QUANT\_BAND\_MIN\_THRESHOLD\_LIST\_PAST,

Similarly,

  SAI\_ARS\_PROFILE\_ATTR\_QUANT\_BAND\_MAX\_THRESHOLD\_LIST\_PAST,

  SAI\_ARS\_PROFILE\_ATTR\_QUANT\_BAND\_MIN\_THRESHOLD\_LIST\_FUTURE,

  SAI\_ARS\_PROFILE\_ATTR\_QUANT\_BAND\_MAX\_THRESHOLD\_LIST\_FUTURE,